README.md

```diff
@@ -64,6 +64,11 @@ any given time, the site may not be compatible with the very latest API version
      bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
      ```
 
+     If you receive an error that `You must install GnuPG to import release team keys` try installing GnuPG with homebrew.
+     ```
+     brew install gnupg
```

Comment:
I found [ASDF documentation](http://asdf-vm.com/guide/getting-started.html#plugin-dependencies) that suggests that `gnupg` is now needed from the start*, so I think we should add it to the end of the long `brew install coreutils automake autoconf openssl libyaml readline libxslt libtool unixodbc` list a few steps above this one.


*the link suggests to `brew install gpg` – it turns out `gpg` is an alias for `gnupg`